### PR TITLE
Improved Python style in importDWG code

### PR DESCRIPTION
**Pull request by vocx-fc**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---
Improved the Pythonic style of the code, particularly the spacing, line length, and adding docstrings to the public functions.

The code was inspected with
```flake8 --ignore=E226,E266,E401,W503 importDWG.py```

### CODE REVIEW

**Comment by berndhahnebach** [comment]
https://forum.freecadweb.org/viewtopic.php?f=23&t=38360

**Comment by drunkwcodes** [comment]
The numpydoc-style docstring parts looked fine.
But the code changes are controversial.

Please split the PRs by seperating docstring changes and non-docstring changes to have better review.

Also it is good to use black or autopep8 formatting tools.

**Comment by vocx-fc** [comment]
I'm not sure which changes are "controversial". This pull request has 5 commits. You can see that the docstrings were added only in the first commit. The rest of the changes are pretty minor, like adding a space after a comma to improve readability. The functionality isn't changed, it's just adding a bit more whitespace and breaking up lines that are too long. I don't think it's hard to follow if you just compare the first commit with the second one, and then see the next ones in sequence.

**Comment by drunkwcodes** [comment]
The PR is actually bad-written and the docstrings are inaccurate.

The codebase formatting had better be done by formatter to avoid next massive re-format.

**Comment by vocx-fc** [comment]
What? What exactly is inaccurate? You really need to provide full answers instead of just disagreeing. This is not a mass reformat.

**Comment by drunkwcodes** [comment]
WHY YOU DROP THE ORIGINAL DOCSTRING???

THE OPTIONAL RETURN TYPES ARE NOT WRITTEN WELL, EITHER!!!

THIS IS NOT A STYLE IMPROVE, IT IS A MESS!!!

**Comment by berndhahnebach** [comment]
@drunkwcodes: FYI there is a disscussion on the forum https://forum.freecadweb.org/viewtopic.php?f=23&t=38360

**Comment by vocx-fc** [comment]
> WHY YOU DROP THE ORIGINAL DOCSTRING???

The original docstrings are like this.
```
def convertToDwg(dxffilename,dwgfilename):
    "converts a DXF file to DWG"
```

That is entirely useless to create automated documentation with Doxygen or Sphinx. See [Doxygen](https://www.freecadweb.org/wiki/Doxygen) (Section "Doxygen with Python code"), and also read pages 3 and 4 of [FEM Workbench Testing - GSoC Proposal](https://forum.freecadweb.org/viewtopic.php?f=18&t=35428&start=20).

Proper documentation of functions need to have raw comments (triple quotes) specifying the input values and return values. This is basic Python convention, compliant with [PEP257](https://www.python.org/dev/peps/pep-0257/), [Documenting Python Code: A Complete Guide](https://realpython.com/documenting-python-code/#docstrings-background), [Re: Python codeformating Draft, Arch in the regard of pep8 etc](https://forum.freecadweb.org/viewtopic.php?p=325457#p325457).
```
def convertToDwg(dxffilename, dwgfilename):
    """Convert a DXF file to a DWG file.

    If the converter is found it is used, otherwise the conversion fails.
    Parameters
    ----------
    dxffilename : str
        The input DXF file
    dwgfilename : str
        The output DWG file
    Returns
    -------
    str
        The same `dwgfilename` file path.
    """
```
I have no idea what you are complaining about. This is done the proper way, in order to produce a complete, automatically generated, [FreeCAD API documentation](https://www.freecadweb.org/api/) that both users and developers can use. Currently a lot of the programming documentation is hosted in the [wiki](https://www.freecadweb.org/wiki/Draft_API) but it is old, obsolete, and nobody will update it. Automatic generation of documentation is 

> THE OPTIONAL RETURN TYPES ARE NOT WRITTEN WELL, EITHER!!!

Which optional return types? YOU NEED TO PROVIDE DETAILS. There are no optional return types in the functions.

**Comment by drunkwcodes** [comment]
The original docstrings are useful, they make your muddy formatting more serious and cover your lame documenting skill.

The details are not more important than the wrong PR title.

**Comment by vocx-fc** [comment]
How is a broken docstring that doesn't conform to any Python standard better than a docstring that complies with the Numpy/SciPy standard style of documentation?

Wrong PR title? Are you seriously complaining about this? If the project managers want to rename the pull request they can do that. That. is. not. important. What is important is the code, which works and is better documented.

I'm not going to reply to you any more because you lack complete sense. It sounds like you are only arguing for the sake of arguing.

**Comment by yorikvanhavre** [comment]
I don't see anything wrong with the docstrings in this PR either...
